cmd/cmd_test.go

```diff
@@ -313,6 +313,13 @@ func (s *cmdSuite) TestExecInCoreSnapUnsetsDidReexec(c *C) {
 	os.Setenv("SNAP_DID_REEXEC", "1")
 	defer os.Unsetenv("SNAP_DID_REEXEC")
 
+	restore := release.MockReleaseInfo(&release.OS{ID: "ubuntu"})
+	defer func() {
+		restore()
+		dirs.SetRootDir(s.fakeroot)
```

Comment:
Did you mean `dirs.SetRootDir("/")`?

Reply:
No, I actually meant to restore it to whatever was set up by the test suite right here: https://github.com/snapcore/snapd/blob/master/cmd/cmd_test.go#L62

Mocking release info slightly alters how the directories are set up, once it's restored SetRootDir() should also restore the directory paths to their original values.

Reply:
+1